### ISSUE TITLE
Unable to sync ttu to Google Drive

### ISSUE THREAD

**Issue by krukgit** [lrorpilla/jidoujisho#194]
Hi! 

I just started using jidoujisho and have been trying to set up sync between ttu in jidoujisho and on https://reader.ttsu.app/ on my PC. I followed the steps described in https://github.com/ttu-ttu/ebook-reader#storage-sources to get the necessary keys. I successfully  connected https://reader.ttsu.app/ to my account, but when I try to do the same in jidoujisho, I get the error:

```
Webpage not available. The webpage at http://localhost:52059/auth?ttu-init-auth=1 could not be loaded because: net::ERR_HTTP_RESPONSE_CODE_FAILURE.
```

Closing this window also shows: `Failure. An Error occured: Window was closed before login`.

Am I doing something wrong, or does jidoujisho not support GDrive sync? I'm using jidoujisho 2.6.3 on Android 13. 
Both http://localhost:52059/auth and http://localhost:52059 are authorised in my project.

**Comment by lrorpilla** [comment]
@krukgit You'll need to use auth.html instead of auth, can you try that and see how you go?

**Comment by krukgit** [comment]
Do you mean I should authorise http://localhost:52059/auth.html redirect URI? Or something else? 
I tried but it had no effect.

![cloud](https://user-images.githubusercontent.com/1926883/236669905-fbc0e89e-d8cc-4312-8d67-41276ca9a4a2.png)

FWIW, I also tried logging into local ttu directly from Chrome by navigating to `localhost:52059`. There, if I try to connect to GDrive I'm getting an error `This localhost page can't be found. No webpage was found for the web address: http://localhost:52059/auth?ttu-init-auth=1`. Changing the URL to auth.html shows me a ttu loading animation but of course by then it would have lost whatever POST params it was trying to pass. Is it possible to somehow configure the local ttu to use `/auth.html`?

**Comment by lrorpilla** [comment]
The app as os definitely supports setting your own GDrive sync, because I've had a user try and succeed. Not sure what's going on here but I'll see what's what.

**Comment by krukgit** [comment]
Thanks. Let me know if you need any additional info from my end.

**Comment by lrorpilla** [comment]
@krukgit Just needed to adjust the updated ッツ I include to point to `auth.html` as well. Will re-release 2.6.4 in a couple minutes. It's possible the website has cached so to see a difference, you should do a fresh install.

**Comment by lrorpilla** [comment]
@krukgit I have re-released 2.6.4 (on arm64, rest are uploading). Can you please try and see if ッツ cloud sync redirection now works for you? I have tested the changes and it redirects fine on an emulator.

**Comment by krukgit** [comment]
I confirmed it works now. Thanks!
